cli/src/config/revsets.toml

```diff
@@ -9,6 +9,7 @@ simplify-parents = "reachable(@, mutable())"
 # suppress symbol resolution error.
 log = "present(@) | ancestors(immutable_heads().., 2) | present(trunk())"
 sign = "reachable(@, mutable())"
+push = "remote_bookmarks(remote=<remote>)..@"
```

Comment:
@yuja what alias hack have you mind to solve the problem here?

Reply:
Naming aside: `'push(remote)' = 'remote_bookmarks(remote=remote)..@'` (i.e. substitute like an alias function)

Reply:
done